### DISCUSSION TITLE
Unsubscribe page/list

### DISCUSSION THREAD

**Discussion by jazer143** [postalserver/postal#2016] [Development discussions]
Hi There,

Seeking your expert advise with regards to email unsubscribe? Is there a request to add an unsubscribe page and/or list in Postal? or anyone here already implemented this in your own Postal setup? since most marketing email providers have this functions and Postal already have the suppression list but has expirations, maybe we can use the same feature for unsubscribe list. Appreciate anyone who can share an idea on this feature. Thank you.

**Comment by willpower232** [comment]
Postal doesn't have a mailing list feature so you would need to implement the unsubscribe feature yourself in whatever system you're using to send messages via Postal.

Suppressions are automatically cleared after a certain time so would not be appropriate for this use case.
